mockwebserver/src/main/java/okhttp3/mockwebserver/SocketPolicy.java

```diff
@@ -31,6 +31,11 @@
  */
 public enum SocketPolicy {
 
+  /**
+  * Shutdown {@link MockWebServer} after writing response.
+  */
```

Comment:
nit: these two lines should be indented by one space

Reply:
+